corehq/apps/app_manager/static/app_manager/js/releases.js

```diff
@@ -398,7 +398,7 @@ hqDefine('app_manager/js/releases.js', function () {
                     } else if (self.lastAppVersion() !== self.currentAppVersion()) {
                         self.actuallyMakeBuild();
                     } else {
-                        window.alert(gettext("No new changes to deploy!"));
+                        window.alert(gettext("No new changes!"));
```

Comment:
something other than an alert box, which is quite jarring. Never thought exactly how this should be shown, but since there is already the "getting latest versions" and "building latest version" loading text, it could probably be a transient message that goes in that same spot.